mmcv/runner/base_runner.py

```diff
@@ -330,6 +331,19 @@ def resume(self,
 
         self._epoch = checkpoint['meta']['epoch']
         self._iter = checkpoint['meta']['iter']
+
+        if 'config' in checkpoint['meta']:
+            temp_file = tempfile.NamedTemporaryFile()
+            config_path = f'{temp_file.name}.py'
```

Comment:
Seems that you just want to get a temporary directory name instead of creating a NamedTemporaryFile.